.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,16 @@
+name: Unit_test
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7
+    - name: Install dependencies
+      run: bundle install
+    - name: Run tests
+      run: rspec ./units/test_helpers/test_image_helper_spec.rb --default-path ./units
+      shell: bash
```

Comment:
Is `shell: bash` line needed? I do not use it in mine workflows and everything seems fine